fk/luae_fs/src/main/java/se/inera/intyg/common/luae_fs/v1/rest/LuaefsModuleApiV1.java

```diff
@@ -121,7 +121,7 @@ public PdfResponse pdf(String internalModel, List<Status> statuses, ApplicationO
             final FkPdfDefinition fkPdfDefinition = builder.buildPdfDefinition(luaefsIntyg, statuses, applicationOrigin,
                 texts, utkastStatus, pdfMinaIntygMarginText);
 
-            return new PdfResponse(PdfGenerator.generatePdf(fkPdfDefinition),
+            return new PdfResponse(PdfGenerator.generatePdf(fkPdfDefinition, CERTIFICATE_FILE_PREFIX),
```

Comment:
I might be wrong but shouldn't the second argument to generatePdf be the file name and not the file prefix?

Reply:
The prefix in this case is the file name.